src/components/timer-ticking.ts

```diff
@@ -26,7 +26,8 @@ export class TimerTicking extends EventManager {
     public isStoneDropped: boolean = false;
```

Comment:
please move to `timer-ticking` folder under `components`
i.e.
`components/timer-ticking/timer-tiking.ts`

also please add at least 1 test case.

so the contents of `timer-ticking` folder will be:
- ts file
- scss file
- spec file